### PR TITLE
Adding documentation for `asset` and `path`.

### PR DESCRIPTION
**Pull request by bobdenotter**

### CODE REVIEW

File: docs/internals/container-service-references.md [hunk 1]
```diff
@@ -95,6 +95,12 @@ URL paths available are:
     "rooturl" => "https://bolt.cm/"
 ```
 
+The paths are available in Twig templates under the deprecated `{{ paths }}`
+variable. Because this variable is depracted, its use is discouraged. Use the
```
Comment (bobdenotter) [inline docs/internals/container-service-references.md]: Fixed.

**Comment by GawainLynch** [comment]
Merged into `release/3.0` via 384cb618016091d3406aa58c731494da0fa8fb54

**Comment by PhillippOhlandt** [comment]
Isn't the asset function only available in 3.1 or 3.2?

**Comment by PhillippOhlandt** [comment]
Anyway, I might start working on the templating section today, in release/3.0

**Comment by GawainLynch** [comment]
https://github.com/bolt/bolt/blob/release/3.0/src/Provider/AssetServiceProvider.php#L27-L30

**Comment by PhillippOhlandt** [comment]
ok, then I don't understand the discussion yesterday.

**Comment by GawainLynch** [comment]
Zero stress … the changes that were 3.1 specific were about back-end. Either way, Bolt 3.0 is no longer supported and 3.1 will EoL in a week or so too. :wink: 

> Anyway, I might start working on the templating section today, in release/3.0

Yeah, exactly why branch targets matter so much to the poor soul(s) merging branches.

Really looking forward to seeing your work land mate!

**Pull request closed without merging**